### PR TITLE
EF-3053 aggregating eip2612 spends

### PR DESCRIPTION
**Pull request by bruscantini**

### CODE REVIEW

**Review by kbihani — APPROVED** [review]
LGTM

File: src/common/engine.js [hunk 1]
```diff
@@ -365,15 +365,25 @@ export class CurrencyEngine {
 
       const { otherParams: otherParamsOld = {} } = edgeTx
       const { otherParams: otherParamsNew = {} } = edgeTransaction
+
+      const oldDestinationAddresses = otherParamsOld.to || [];
+      const newDestinationAddresses = otherParamsNew.to || [];
+      const shouldAddEip2612Spend = newDestinationAddresses.length &&
```
Comment (MarioCerdan) [inline src/common/engine.js]: wondering if we can lock it to just this usdc gasless wallet. 
maybe we can pass to this Edge plugin the Biconomy forwarder address and check that the tx goes to it?

Comment (bruscantini) [reply src/common/engine.js]: I could not find an easy way to "pass it to the plugin", but if we're gonna do that, we can just hardcode it here and just make a new build whenever we have to update it. I think we're good like this, tho.

Comment (MarioCerdan) [reply src/common/engine.js]: dale, lets do it

Comment (bruscantini) [reply src/common/engine.js]: Do the hardcoding? Ok, then we'll have to wait for biconomy to give us the feeReceiver address.

Comment (MarioCerdan) [reply src/common/engine.js]: can we get the destination of the tx ("Interacted With (To):" in etherscan) ? 

https://etherscan.io/address/0xc924e1bfb69acf04f74e279162231da107413b49

this is our TransferHandlerCustom.

Comment (bruscantini) [reply src/common/engine.js]: No, sir. The "interacted with (to)" is not available in the EdgeTransaction object:
```
{
  "txid": "0x5c7906b03215854cdcf72b55428371b289ce412f27c7bf28dd907ab0b8742274",
  "date": 1626902169,
  "currencyCode": "USDC",
  "blockHeight": 12872146,
  "nativeAmount": "-6591560",
  "networkFee": "0",
  "ourReceiveAddresses": [],
  "signedTx": "",
  "otherParams": {
    "from": [
      "0x67ba9f45e61b6ca6d4f608bbea9927bbaae3388f"
    ],
    "to": [
      "0x71e19bf5de5b5783e99f1046b5c27e6f69cdff41"
    ],
    "gas": "200000",
    "gasPrice": "21000000000",
    "gasUsed": "138104",
    "cumulativeGasUsed": "10540433",
    "errorVal": null,
    "tokenRecipientAddress": null
  }
}
```

Comment (MarioCerdan) [reply src/common/engine.js]: i see...argh. 

then we can simply use the currencyCode ? hardcode it, it needs to be usdc and done....

Comment (bruscantini) [reply src/common/engine.js]: donezo

**Pull request merged**